recipes/ignition-utils/all/conanfile.py

```diff
@@ -0,0 +1,138 @@
+             self, "ign-utils-ignition-utils{}_{}".format(version_major, self.version),
+             self._source_subfolder
+            )
+
+    def _configure_cmake(self):
+        if self._cmake is not None:
+            return self._cmake
+        self._cmake = CMake(self)
+        self._cmake.definitions["BUILD_TESTING"] = False
+        #self._cmake.definitions["IGN_UTILS_VENDOR_CLI11"] = True
+        self._cmake.definitions["CMAKE_FIND_DEBUG_MODE"] = "1"
+        self._cmake.configure()
+        return self._cmake
+
+    def build(self):
+        for patch in self.conan_data.get("patches", {}).get(self.version, []):
+            tools.patch(**patch)
+        cmake = self._configure_cmake()
+        cmake.build()
+
+    def package(self):
+        self.copy("LICENSE", dst="licenses", src=self._source_subfolder)
+        cli_header_src = os.path.join(self._source_subfolder, "cli", "include")
+        if int(tools.Version(self.version).minor) is 0:
+            cli_header_src = os.path.join(cli_header_src, "ignition", "utils", "cli")
+        else:
+            cli_header_src = os.path.join(cli_header_src, "external-cli", "ignition", "utils", "cli")
+        self.copy("*.hpp", src=cli_header_src,
+                     dst="include/ignition/utils1/ignition/utils/cli")
+        cmake = self._configure_cmake()
+        cmake.install()
+        tools.rmdir(os.path.join(self.package_folder, "share"))
+        tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
+        tools.rmdir(os.path.join(self.package_folder, "lib", "cmake"))
+
+        # Remove MS runtime files
+        for dll_pattern_to_remove in ["concrt*.dll", "msvcp*.dll", "vcruntime*.dll"]:
+            tools.remove_files_by_mask(os.path.join(self.package_folder, "bin"), dll_pattern_to_remove)
+
+    def package_info(self):
+        version_major = tools.Version(self.version).major
+        self.cpp_info.names["cmake_find_package"] = "ignition-utils{}".format(version_major)
+        self.cpp_info.names["cmake_find_package_multi"] = "ignition-utils{}".format(version_major)
+
+        # cmake_find_package filename: ignition-utils-config.cmake
+        self.cpp_info.components["libignition-utils"].libs = ["ignition-utils{}".format(version_major)]
+        self.cpp_info.components["libignition-utils"].includedirs.append("include/ignition/utils{}".format(version_major))
+        self.cpp_info.components["libignition-utils"].names["cmake_find_package"] = "ignition-utils{}".format(version_major)
+        self.cpp_info.components["libignition-utils"].names["cmake_find_package_multi"] = "ignition-utils{}".format(version_major)
+        self.cpp_info.components["libignition-utils"].names["pkg_config"] = "ignition-utils{}".format(version_major)
+        self.cpp_info.components["libignition-utils"].requires = ["cli11::cli11"]
+        self.cpp_info.components["libignition-msgs"].requires.append("doxygen::doxygen")
+        self.env_info.LD_LIBRARY_PATH.extend([
```

Comment:
are there subfolders on library directory? Otherwise, it should not be needed .

Reply:
removed